docs/zz_generated.seed.yaml

```diff
@@ -81,6 +81,7 @@ spec:
         # EnforcePodSecurityPolicy enforces pod security policy plugin on every clusters within the DC,
         # ignoring cluster-specific settings
         enforcePodSecurityPolicy: false
+        fake: null
```

Comment:
This file needs to be regenerated once the `omitgenyaml` tags are back in place.

Reply:
All the docs changes have been made by a `./hack/update-docs.sh` run after a rebase to master.